src/api_proxy/service_control/request_builder.cc

```diff
@@ -1364,6 +1366,19 @@ Status RequestBuilder::ConvertCheckResponse(
         check_response.check_info().consumer_info().project_number());
   }
 
+  if (check_response.check_info().consumer_info().consumer_number() > 0) {
+    check_response_info->consumer_number = std::to_string(
+        check_response.check_info().consumer_info().consumer_number());
+  }
+
+  if (check_response.check_info().consumer_info().type() !=
+      CheckResponse_ConsumerInfo_ConsumerType::
+          CheckResponse_ConsumerInfo_ConsumerType_CONSUMER_TYPE_UNSPECIFIED) {
+    check_response_info->consumer_type =
+        CheckResponse_ConsumerInfo_ConsumerType_Name(
```

Comment:
Is this from proto generated code?   is this a exposed public functions or just internal implementation.  if latter,  proto team can remove it anytime, we will break.

Reply:
It is generated. 

https://github.com/googleapis/googleapis/blob/8c53b2cb792234354c13336ac7daee61333deade/google/api/servicecontrol/v1/service_controller.proto#L122

Reply:
Follow the convention of nestedProtoDefinition + _name